DominicanWhoCodes.Blazor/wwwroot/data/developers.json

```diff
@@ -193,6 +193,15 @@
     "twitter":"https://twitter.com/enriquecapellan"
   },
   {
+    "name": "Angel A Perez",
+    "initials": "AAP",
+    "image": "data/devs-pics/angel-perez.jpg",
+    "summary": "Backend developer always looking for a new challenge.",
+    "skills": "C#, C++ and Node.js",
+    "linkedin": "https://www.linkedin.com/in/angel-perez-severino/",
+    "github": "https://github.com/aperez011",
+    "twitter": "https://twitter.com/ApSeverinol011"
+  },
     "name": "Yhorby Matias",
```

Comment:
Seems like you are breaking something here, try to add yourself at the end of the array

Reply:
I fixed the issue in the file